starlette/authentication.py

```diff
@@ -95,7 +97,7 @@ def sync_wrapper(*args: typing.Any, **kwargs: typing.Any) -> Response:
 
             return sync_wrapper
 
-    return decorator
+    return decorator  # type: ignore
```

Comment:
```suggestion
    return decorator  # type: ignore[return-value]  # type checkers can't verify the type of the wrapper
```

Reply:
no need the comment, just the return-value thingy